docs/conf.py

```diff
@@ -160,11 +160,19 @@
 myst_heading_anchors = 3
 
 version_string = '.'.join(str(x) for x in __version_tuple__[:3])
+python_version = '3.9'
+python_version_range = '3.8–3.10'
+python_minimum_version = '3.8'
 
 myst_substitutions = {
    "napari_conda_version": f"`napari={version_string}`",
    "napari_version": version_string,
- }
+   "python_version": python_version,
+   "python_version_range": python_version_range,
+   "python_minimum_version": python_minimum_version,
+   "python_version_code": f"`python={python_version}`",
+   "conda_env_python_version": f"```sh\nconda create -y -n napari-env -c conda-forge python={python_version}\nconda activate napari-env\n```",
```

Comment:
MAybe change variable name to `conda_create_instruction`?

Reply:
I like that.
The other option is to move that substitution inline in the .md, instead of in conf.py.
I think something like this would work:
```
{{ "```sh\nconda create -y -n napari-env -c conda-forge python=PYTHON_VERION}\nconda activate napari-env\n```".replace(PYTHON_VERION, python_version }}
```
But those conda instructions are reused  3 times...

Reply:
But substitution declaration in the markdown file is mainly for things specific for a given file.

Reply:
Agreed....
...and done.